tsconfig.json

```diff
@@ -8,7 +8,8 @@
 			"node-fetch",
 			"discord.js",
 			"klasa",
-			"klasa-dashboard-hooks"
+			"klasa-dashboard-hooks",
+			"@klasa/settings-gateway"
```

Comment:
Please don't. TypesSript resolves all typings in `node_modules/@types/**` and `node_modules/**/package.json -> types or typings fields`

https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#types-typeroots-and-types

Reply:
What it says it does and what it actually does, is bullshit.

Reply:
Not a single time in the past year of writing TypeScript have I ever needed the types field. The single only projects that I've seen use it is Skyra, fs-nextra and other dirigeants projects. Have you tried removing it and seeing if everything still compiles fine?

And if your claim really is true you're perfectly free to re-write it: https://github.com/microsoft/TypeScript-Website

Reply:
Of course I have, otherwise I wouldn't be able to make such assertion.
Either way this case wasn't needed, so I removed it in the following commit.